proto/profile.proto

```diff
@@ -113,7 +118,7 @@ message Label {
   // Should only be present when num is present.
   // Specifies the units of num.
   // Use arbitrary string (for example, "requests") as a custom count unit.
-  // If no unit is specified, consumer may apply heuristic to deduce the unit.
+  // If no unit is specificed, consumer may apply heuristic to deduce the unit.
```

Comment:
The change looks like it introduces a typo.

Reply:
Fixing here and separately in profile.proto